tests/EnergonTest.php

```diff
@@ -102,7 +102,7 @@ public function testInvalidPrimeProvided()
     {
         $this->expectException(InvalidPrimeException::class);
 
-        Energon::generate(2);
+        Energon::generate(4);
```

Comment:
Two is a valid prime number ([wiki](https://en.wikipedia.org/wiki/Prime_number)). I don't understand why it used to cause an error.